insights/parsers/sys_module.py

```diff
@@ -101,3 +131,45 @@ class VHostNetZeroCopyTx(XModUseBlkMq):
 
     """
     pass
+
+
+@parser(Specs.lpfc_max_luns)
+class LpfcMaxLUNs(MaxLUNs):
+    """
+    This file `/sys/module/lpfc/parameters/lpfc_max_luns` shows the max LUN number
+    supported by lpfc driver.
+
+    Examples::
```

Comment:
`Examples:` should only have 1 colon.  See the [recommended style guide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for more information.